### PR TITLE
symbiosis adds usdt on ethereum and syUSDC on telos chain

### PR DESCRIPTION
**Pull request by allush**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Please enable "Allow edits by maintainers" while putting up the PR.
3. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Github org/user (Optional, if your code is open source, we can track activity):

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/symbiosis-finance exports TVL: 
        
 
 
    mantle                    105.37 k
    era                       874.18 k
    optimism                  79.69 k
    linea                     180.43 k
    base                      114.24 k
    kava                      0
    bsc                       468.48 k
    polygon                   448.93 k
    avax                      110.30 k
    ethereum                  694.52 k
    boba_avax                 1.818174
    arbitrum                  118.79 k
    aurora                    4.326675995664
    arbitrum_nova             48.93 k
    boba                      150.86 k
    polygon_zkevm             29.15 k
    boba_bnb                  189.36 k
    telos                     172.32394313376003
    
    total                    3.61 M

**Comment by allush** [comment]
@g1nt0ki `syUSDC` is a new officialy bridged USDC from Ethereum to Telos instead of Multichain USDC. 
How to start calculation TVL of syUSDC?

File: projects/symbiosis-finance/config.js [hunk 1]
```diff
@@ -56,6 +57,7 @@ module.exports = {
       name: 'telos',
       tokens: [
         ADDRESSES.telos.USDC,
+        ADDRESSES.telos.syUSDC,
```
Comment (g1nt0ki) [inline projects/symbiosis-finance/config.js:60]: what is this syUSDC? is it symbiosis USDC?

**Comment by g1nt0ki** [comment]
> @g1nt0ki `syUSDC` is a new officialy bridged USDC from Ethereum to Telos instead of Multichain USDC. How to start calculation TVL of syUSDC?

Sorry, but can you share some info to verify the syUSDC address, I tried  googling but unable to find any info, once verified, we will price syUSDC on our server and merge this PR

**Pull request merged**